source/mainnet/smart-contracts/tutorials/sponsoredTransactions/index.rst

```diff
@@ -0,0 +1,54 @@
+.. include:: ../../../variables.rst
+.. _sponsoredTransactions:
+
+.. image:: sponsoredTransactions.png
+   :width: 40%
+   :align: center
+   :alt: An eSealing dApp
+
+=============================
+A Sponsored Transactions dApp
+=============================
+
+In this tutorial, you are going to get familiar with how to implement a sponsored transaction mechanism in a smart contract, how to sign messages in a wallet and verify the signature in the smart contract,
+how to send a transaction from a backend server to the blockchain, and how to connect your |mw-gen2| via walletConnect to the frontend.
+
+The dApp consists of three components that you will explore as part of this tutorial:
+
+- A `smart contrat` that verifies signatures.
+
+- A `frontend` that requests the user to sign messages.
+
+- A `backend` that sends transactions to the blockchain.
```

Comment:
Perhaps clarify here that this is the "sponsoring" part. This backend is the sponsor.